### PR TITLE
ci: Format black

### PR DESCRIPTION
**Pull request by saschahofmann**
ci: ran black over the codebase to be in accordance with the main repo and add CI step

### CODE REVIEW

**Comment by tswast** [comment]
Thanks! Let's make sure we add a check that black has run in our CI config at https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/main.yml before merging this.

**Comment by saschahofmann** [comment]
@tswast finally added the CI step!

File: tests/system/test_client.py [hunk 1]
```diff
@@ -722,9 +708,9 @@ def test_numeric_sum(numeric_table):
 
 def test_boolean_casting(alltypes):
     t = alltypes
-    expr = t.groupby(k=t.string_col.nullif('1') == '9').count()
-    result = expr.execute().set_index('k')
-    count = result['count']
+    expr = t.groupby(k=t.string_col.nullif("1") == "9").count()
+    result = expr.execute().set_index("k")
+    count = result["count"]
     assert count.loc[False] == 5840
```
Comment (tswast) [inline tests/system/test_client.py:714]: I see that this line is failing in the system tests, but must be due to a behavior change in pandas, not this PR.

Comment (tswast) [reply tests/system/test_client.py:714]: Filed https://github.com/ibis-project/ibis-bigquery/issues/89.

I don't think it should block this PR.

**Comment by tswast** [comment]
Thanks for the contribution, @saschahofmann !

**Pull request merged**